yb_migrate/src/migration/postgres.go

```diff
@@ -314,7 +309,13 @@ func getMappingForTableNameVsTableFileName(dataDirPath string) map[string]string
 			continue
 		} else if parts[3] == "TABLE" && parts[4] == "DATA" {
 			fileName := strings.Trim(parts[0], ";") + ".dat"
-			fullTableName := parts[5] + "." + parts[6]
+			schemaName := parts[5]
+			tableName := parts[6]
+			if nameContainsCapitalLetter(tableName) {
+				// Surround the table name with double quotes.
+				tableName = fmt.Sprintf("\"%s\"", tableName)
```

Comment:
one question here, wouldn't the tableName in case of case sensitive already be inside double quotes? For Postgres tableList, we are just parsing the .sql files for table names not querying the system tables right?

Reply:
The table-names in the output of the `pg_restore -l DATA_DIR` are not enclosed in double quotes:
```
3783; 0 19602 TABLE DATA public Artist postgres
3782; 0 19597 TABLE DATA public Album postgres
3785; 0 19612 TABLE DATA public Employee postgres
3784; 0 19607 TABLE DATA public Customer postgres
3786; 0 19617 TABLE DATA public Genre postgres
3787; 0 19622 TABLE DATA public Invoice postgres
3789; 0 19632 TABLE DATA public MediaType postgres
3792; 0 19647 TABLE DATA public Track postgres
3788; 0 19627 TABLE DATA public InvoiceLine postgres
3790; 0 19637 TABLE DATA public Playlist postgres
3791; 0 19642 TABLE DATA public PlaylistTrack postgres
```
That's why we need to explicitly enclose the table-names in double quotes if they contain any upper case letter(s).

Reply:
ohh ya got it, i forgot not we have updated the coding using pg_restore for table list